### PR TITLE
[14.0][ADD] stock_quant_reservation_info

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
This module allows to look who is reserving the products and check the following picking.

@ForgeFlow

### CODE REVIEW

**Comment by JuanyDForgeflow** [comment]
@DavidJForgeFlow Functional review is fine, I was able to see the field and button after refreshing the page but not when I enter directly from the qty on hand button, is that ok?

image

**Comment by DavidJForgeFlow** [comment]
> @DavidJForgeFlow Functional review is fine, I was able to see the field and button after refreshing the page but not when I enter directly from the qty on hand button, is that ok?
> 
> image
Thanks @JuanyDForgeflow , there are 2 differents views and I only was changing one of them! Already fixed up!

**Review by JuanyDForgeflow — APPROVED** [review]

File: stock_quant_reservation_info/models/stock_move_line.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+from odoo import models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.move.line"
+
+    def action_view_picking_from_reserved(self):
+        action = self.env.ref("stock.action_picking_tree_all")
+        result = action.sudo().read()[0]
```
Comment (rousseldenis) [inline stock_quant_reservation_info/models/stock_move_line.py]: Use _for_xml_id instead.

File: stock_quant_reservation_info/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+This module allows to look who is reserving the products and check the following picking.
```
Comment (rousseldenis) [inline stock_quant_reservation_info/readme/DESCRIPTION.rst]: Who ? Could you be a little bit more accurate ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

File: stock_quant_reservation_info/models/stock_move_line.py (hunk 1 above)
Comment (DavidJForgeFlow) [reply stock_quant_reservation_info/models/stock_move_line.py]: Changed, thanks!

File: stock_quant_reservation_info/readme/DESCRIPTION.rst (hunk 2 above)
Comment (DavidJForgeFlow) [reply stock_quant_reservation_info/readme/DESCRIPTION.rst]: Also changed, thanks again!

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**